packages/@aws-cdk/aws-athena/package.json

```diff
@@ -68,14 +68,18 @@
     "pkglint": "1.18.0"
   },
   "dependencies": {
-    "@aws-cdk/core": "1.18.0"
```

Comment:
Leave the `@aws-cdk/core` dependency (this module uses a bunch of things from it, like `Construct`).

Reply:
It was just pushed down a line so the dependencies are in alphabetical order